### PR TITLE
[DEVOPS] Add screenfetch, help menus, and aliases to chrommunal setup scripts

### PR DESCRIPTION
**Pull request by onibrow**
screen fetch and help menu

### CODE REVIEW

File: DevOps/frankfurter/chrommunal/bashrc [hunk 1]
```diff
@@ -6,3 +6,11 @@ fi
 alias dawn='sudo startxiwi -n trusty /opt/dawn/dawn'
 alias term='sudo enter-chroot'
 alias ubuntu='sudo startunity -X xorg'
+alias screenfetch='/./usr/local/bin/screenfetch-dev -E'
```
Comment (ArjunSridhar1) [inline DevOps/frankfurter/chrommunal/bashrc]: Why is there a '/' before the './'? 
Is this correct?

Comment (onibrow) [reply DevOps/frankfurter/chrommunal/bashrc]: Eeerrrr its been too long, I'll look at this sometime during the Summer.

**Review by ArjunSridhar1 — APPROVED** [review]

File: DevOps/frankfurter/chrommunal/bashrc (hunk 1 above)
Comment (rmandla) [reply DevOps/frankfurter/chrommunal/bashrc]: I don't think there is supposed to be a /

Comment (rmandla) [reply DevOps/frankfurter/chrommunal/bashrc]: nvm, it's fine. but i don't think that file is executable, so you got to run it with bash. Also, when i tried to run it on my computer, I had to use sudo. Don't know if this is true for chromebooks

Comment (kevinji) [reply DevOps/frankfurter/chrommunal/bashrc]: If `/` should be there, the `./` is definitely redundant.

**Pull request merged**